cmake/evt-core_build.cmake

```diff
@@ -26,4 +26,5 @@ macro(make_exe proj_name sources)
 
     # Link the EVT-core library
     target_link_libraries(${proj_name} PUBLIC EVT)
+    target_link_libraries(${proj_name} PUBLIC threadx)
```

Comment:
Why do we need this but not an equivalent call for the CANopen stack

Reply:
This is probably what we needed to do with the CANOpen stack instead of all the modifications we made to the original library.

Reply:
Do you think it's worth undoing what we did and switching over to this method, so it's easier to work with next time we update the CANopen stack?

Reply:
Also, @DiegoLHendrix, you should update the comment on line 27